### PR TITLE
Metadata: isaaclevin.presencelight version 2.0.49

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21821)

### CODE REVIEW

File: manifests/i/isaaclevin/presencelight/2.0.49/isaaclevin.presencelight.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: isaaclevin.presencelight
+PackageVersion: 2.0.49
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/2.0.49/isaaclevin.presencelight.installer.yaml:4]:
```suggestion
PackageVersion: 2.0.49.0
```

File: manifests/i/isaaclevin/presencelight/2.0.49/isaaclevin.presencelight.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: isaaclevin.presencelight
+PackageVersion: 2.0.49
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/2.0.49/isaaclevin.presencelight.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 2.0.49.0
```

File: manifests/i/isaaclevin/presencelight/2.0.49/isaaclevin.presencelight.yaml [hunk 3]
```diff
@@ -1,19 +1,8 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
 PackageIdentifier: isaaclevin.presencelight
 PackageVersion: 2.0.49
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/2.0.49/isaaclevin.presencelight.yaml:4]:
```suggestion
PackageVersion: 2.0.49.0
```

File: manifests/i/isaaclevin/presencelight/2.0.49/isaaclevin.presencelight.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: appx
+  Scope: machine
+  InstallerUrl: https://github.com/isaacrlevin/PresenceLight/releases/download/v2.0.49/PresenceLight.Package_2.0.49.0_x86.appxbundle
+  InstallerSha256: 9697EBC546131A75D6E022E19B0C83C55DA6F79967C11CE6FE5CEDF80D0CC379
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/2.0.49/isaaclevin.presencelight.installer.yaml:16]:
```suggestion
  InstallerSha256: 9697EBC546131A75D6E022E19B0C83C55DA6F79967C11CE6FE5CEDF80D0CC379
  SignatureSha256: 530A32D48BB9AF53C3DD50F6F406007FA5D07FC5394483396458C281C63C562A
  PackageFamilyName: 37828IsaacLevin.197278F15330A_jvewcxq8vj8qt
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/i/isaaclevin/presencelight/2.0.49/isaaclevin.presencelight.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: appx
+  Scope: machine
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/2.0.49/isaaclevin.presencelight.installer.yaml:14]:
```suggestion
MinimumOSVersion: 10.0.17134.0
InstallModes:
- interactive
- silent
Installers:
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: appx
  Scope: user
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
cc @denelon @palenshus: "Azure-Pipeline-Passed" label needs to be added to this PR.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**